libkbfs/quota_usage.go

```diff
@@ -90,24 +100,25 @@ func (q *EventuallyConsistentQuotaUsage) getAndCache(
 // 3) If the age of cached data is more than tolerance, a blocking RPC is
 // issued and the function only returns after RPC finishes, with the newest
 // data from RPC. The RPC causes cached data to be refreshed as well.
-func (q *EventuallyConsistentQuotaUsage) Get(ctx context.Context,
-	tolerance time.Duration) (usageBytes, limitBytes int64, err error) {
+func (q *EventuallyConsistentQuotaUsage) Get(
```

Comment:
Please update the comment to explain the different tolerances (and remove the "half" stuff).